.readme/template.md

```diff
@@ -4,7 +4,7 @@
 
 Collection of publicly available IPTV channels from all over the world.
 
-Internet Protocol television (IPTV) is the delivery of television content over Internet Protocol (IP) networks.
+Internet Protocol television (IPTV) is the delivery of television content over Internet Protocol (IP) networks. This repository has a community database for compile into organized and ready-to-use playlists the available IPTV stations, regardless of their local or international scope.
```

Comment:
```suggestion
Internet Protocol television (IPTV) is the delivery of television content over Internet Protocol (IP) networks. This repository has a community database that compiles into organized and ready-to-use playlists the available IPTV stations, regardless of their local or international scope.
```

Reply:
I changed the sentence to one you suggested @LaneSh4d0w.